package/gluon-mesh-vpn-tunneldigger/luasrc/lib/lua/gluon/mesh-vpn/proto/tunneldigger.lua

```diff
@@ -0,0 +1,33 @@
+local uci = require("simple-uci").cursor()
+
+local site = require 'gluon.site'
```

Comment:
This is a small thing left regarding code-quality;
the require-statements with their used brackets and quotes are somewhat inconsistent.
But other than that the code looks fine now.